scripts/install.js

```diff
@@ -55,6 +55,13 @@ console.log('Installing & activating plugins ...');
 composer('update', config.paths.container.app);
 installPluginsDependencies(config);
 
+/**
+ * Install development requirements
+ */
+console.log('Installing development requirements ...');
+run('cp .wp-env.override.json.dist .wp-env.override.json');
```

Comment:
can we check if this file exist so we don't override some existing configuration ?

Reply:
I added a `-n` flag here. 

```
-n, --no-clobber
        do not overwrite an existing file
```